DARPA-ASKEM/terarium
# dataset column information redesign

# Description

* Enriching columns occurs reactively
* Column information can now be edited and saved similarly to the model parts
* I am only showing one concept now even though an array of them are generated after enrichment, designs should be updated
* I am not using that `enrichDataset()` in the dataset util - doesn't seem to do anything helpful now

https://github.com/user-attachments/assets/4f9eccfe-d8b3-4410-b0aa-1043c97711c9

### Testing
* Add a new dataset
* Enrich it
* Make some edits and click the save button
* See if its saved
